docker-for-mac/kubernetes-integration.md

```diff
@@ -0,0 +1,37 @@
+---
+description: Docker for Mac and Kubernetes
+keywords: mac, kubernetes, edge, beta, tutorial
+redirect_from:
+- /mackit/kubernetes/
+title: Docker for Mac Kubernetes integration
+---
+
+Docker for Mac has a beta program that includes a full Kubernetes cluster, starting from the Edge 17.12 release.
+To be enrolled in the beta program, you can register [here](https://beta.docker.com/)
+
+## Activating Kubernetes
+
+If you have been enrolled in the beta program, once logged in with your dockerid, you will see a new "Kubernetes" tab in the preference pane : 
```

Comment:
pane:

Reply:
You don't actually need to be logged in.